docs/learn/about-iota/an-introduction-to-iota.md

```diff
@@ -1,87 +1,73 @@
 
 ## **Consensus in the Tangle**
 
-In the Tangle, messages require no fees because the network has no miners or stakers.
-In the Tangle, PoW is not used to secure the network. Instead, PoW is used only to discourage spam messages.
-To reach a consensus, all IOTA nodes validate messages and use different functions alongside messages in their confirmation.
-Currently, messages will only be considered valid if they have a reference to a milestone. These milestones are issued by a special network node - the Coordinator.
+In the Tangle, messages require no fees because the network has no miners or stakers. In the Tangle, PoW is not used to secure the network. Instead, PoW is used only to discourage spam messages. To reach a consensus, all IOTA nodes validate messages and use different functions alongside messages in their confirmation. Currently, messages will only be considered valid if they have a reference to a milestone. These milestones are issued by a special network node – the Coordinator.
 
 ## **The coordinator**
 
 The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. This topic describes how nodes use milestones to determine which messages are confirmed.
 
- - The Coordinator is temporary. We will transition into removing the Coordinator: Coordicide in the upcoming IOTA 2.0 network. Below is how the Coordinator is currently deployed within IOTA.
+- The Coordinator is temporary. We will transition into removing the Coordinator: Coordicide in the upcoming IOTA 2.0 network. The following text describes how the Coordinator is currently deployed within IOTA.
 
 Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
 
-To allow the nodes recognizing the milestones, all nodes that participate in the same IOTA network are configured with the Merkle root address of a Coordinator that they trust to confirm messages. Using this address, nodes can validate the signatures in milestones to verify whether the trusted Coordinator signs them.
+To allow the nodes to recognize the milestones, all nodes that participate in the same IOTA network are configured with the Merkle root address of a Coordinator that they trust to confirm messages. Using this address, nodes can validate the signatures in milestones to verify whether the trusted Coordinator signs them.
 
 To make sure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones at regular intervals. This way, nodes can compare the indexes of their milestones to check whether they are synchronized with the rest of the network.
 
-[![coordinator](/img/learn/milestones.gif)](/img/learn/milestones.gif)
+![coordinator](/img/learn/milestones.gif)
```

Comment:
```suggestion
[![coordinator](/img/learn/milestones.gif)](/img/learn/milestones.gif)
```
This link structure is needed to make images open in a new tab on click to enlarge them